roxctl/scanner/uploaddb/uploaddb_test.go

```diff
@@ -0,0 +1,15 @@
+package uploaddb
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestScannerUploadDbCommandFail(t *testing.T) {
+	t.Run("non existing filename", func(t *testing.T) {
+		cmdNoFile := scannerUploadDbCommand{filename: "non-existing-filename"}
+
+		require.EqualError(t, cmdNoFile.uploadDd(), "could not open file: open non-existing-filename: no such file or directory")
```

Comment:
You can access it by doing the following:
```go
import "github.com/pkg/errors"
import "io/fs"


err := cmdNoFile.uploadDd()
require.Error(t, err)
err = errors.Cause(err) // errors.WithStack implements Cause interface. This will return the underlying error.
// Afterwards we can check the error type, in our case it would be fs.ErrNotExist
assert.ErrorIs(t, err, fs.ErrNotExist)
```

Reply:
👍 ✅